### ISSUE TITLE
Consider not showing "can't find translation" if there's no translation

### ISSUE THREAD

**Issue by Tyrrrz** [mattpocock/ts-error-translator#46]
Example:

![image](https://user-images.githubusercontent.com/1935960/166475380-841f5f1a-07ab-4ca5-8eb5-6cc195d35e08.png)

The original error is succinct and understandable enough. The "Can't find translation" section, along with bold and large text, needlessly steals attention from something that doesn't necessarily require a translation. I would prefer if this message didn't show up in such cases or, if it did, was smaller to reduce noise-to-info ratio.

**Comment by mattpocock** [comment]
Would you prefer:

```
Cannot find namespace 'GatsbyTypes'.

Contribute a translation for #2503
```

I think that makes clear what's going on while being relatively economical in terms of space.

**Comment by Tyrrrz** [comment]
Yeah, that would be great. My main issue is that my eyes instinctively wander towards all the big bold text, while the important (and already useful) bit is the original error.

Also, I'm not sure if it's possible in VS Code, but it would be nice if you could make the `Contribute a translation for #2503` use a smaller font, like it is possible to do in Markdown with `<sup>` or `<sub>`:

<sub>Contribute a translation for #2503</sub>

☝

But otherwise, great work! Can't wait to test it out on more complex type errors that leave me scratching my head for 15 minutes. 😀

**Comment by mattpocock** [comment]
Closing, added this on main.

**Comment by mattpocock** [comment]
Example:

![image](https://user-images.githubusercontent.com/28293365/166484445-8a56a88a-d5df-4dbd-9ba0-fbdf723500c8.png)
